app/destination/page.js

```diff
@@ -1,54 +1,92 @@
 "use client";
 
-import { useState } from 'react';
+import { useState } from "react";
 
-import styles from '@/components/destination/destination.module.css';
-import { AddWishlistItem } from '@/components/destination/AddWishlistItem';
+import styles from "@/components/destination/destination.module.css";
+import { PlanetWishlistItem } from "@/components/destination/PlanetWishlistItem";
+import { AddWishlistItem } from "@/components/destination/AddWishlistItem";
 
-// TASK - React 1 week 2
-// Move this to its own file
-const PlanetWishlistItem = ({
+const PlanetCard = ({
   name,
-  onRemove,
+  description,
   thumbnail,
+  isSelected,
+  onAddOrRemovePlanet,
 }) => {
   return (
-    <div className={styles.wishlistItem}>
-      <img className={styles.wishlistItemThumbnail} src={thumbnail} alt="" />
-      <b>{name.toUpperCase()}</b>
-      <button onClick={onRemove}>remove</button>
+    <div className={styles.planetCard}>
+      <img className={styles.planetThumbnail} src={thumbnail} />
+      <div className={styles.planetDescription}>
+        <h2>
+          {name} {isSelected ? " - SELECTED" : ""}
+        </h2>
+        <p>{description}</p>
+      </div>
+      <button
+        className="roundButton"
+        onClick={() => onAddOrRemovePlanet(name, 0)}
```

Comment:
```suggestion
        onClick={() => onAddOrRemovePlanet(name)}
```
The card should not be aware that it is in a list. It will make your component more reusable.

Also, this code is buggy because you always use the same index (`0`), meaning that it will alwaysremove the first card.

Reply:
Thank you, got it!